source/includes/action.md

```diff
@@ -56,7 +54,7 @@ State | Description
 ----- | -----------
 In progress | The action needed asynchronous execution and is being performed
 Success | The action was executed successfully
-Failed | There was an issue when the action was being performed. Check the logs for more information.
+Failed | There was an issue when the action was being performed. Check the logs for more information (missing pointer for tutum-live doc).
```

Comment:
@fermayo @Siedrix I need to update this with the link to the tutum-live action logs documentation. Let me know when it is ready